cfgov/unprocessed/apps/mmt-cost-of-credit-tool/.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
We should delete the `.vscode` directory from git.  Don't want to include local settings files.